packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -885,6 +890,22 @@ export class GarbageCollector implements IGarbageCollector {
             this.runtime.deleteUnusedRoutes(gcResult.deletedNodeIds);
         }
 
+        // If we are running in GC tombstone mode, tombstone objects for unused routes. This enables testing scenarios
+        // involving access to "deleted" data without actually deleting the data from summaries.
+        if (this.tombstoneMode) {
```

Comment:
Note that if testMode as well as tombstone mode is set, this can result in an error because the data store being tombstone will be deleted by the test mode code above.

Reply:
Updated